profiles/common/modules/features/nexteuropa_trackedchanges/nexteuropa_trackedchanges.features.inc

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * @file
+ * nexteuropa_trackedchanges.features.inc
+ */
+
+/**
+ * Implements hook_ctools_plugin_api().
+ */
+function nexteuropa_trackedchanges_ctools_plugin_api($module = NULL, $api = NULL) {
+  if ($module == "strongarm" && $api == "strongarm") {
+    return array("version" => "1");
+  }
+}
```

Comment:
As explained the other day, It's weird to have this hook in this file... but ok.